src/__tests__/user.spec.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable node/no-unpublished-require */
+const mongoose = require('mongoose');
+const supertest = require('supertest');
+const app = require('../app');
+const UserModel = require('../models/user');
+const connectDatabase = require('../db/connection');
```

Comment:
in the auth test we used `db` instead of `connectDatabase`, so we should stick to one convention.

Reply:
I changed the one in `auth.spec.js` as well: `connectDatabase` is more self-explanatory. Can be changed, but it did already pass a couple of reviews without any issue.